### ISSUE TITLE
v1.11 errors when loading REPL and Pkg

### ISSUE THREAD

**Issue by vpuri3** [JuliaLang/julia#56111]
```julia
[vp@MBP dev]:julia --startup-file=no
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.11.0 (2024-10-07)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> versioninfo()
Julia Version 1.11.0
Commit 501a4f25c2b (2024-10-07 11:40 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: macOS (arm64-apple-darwin22.4.0)
  CPU: 10 × Apple M2 Pro
  WORD_SIZE: 64
  LLVM: libLLVM-16.0.6 (ORCJIT, apple-m2)
Threads: 1 default, 0 interactive, 1 GC (on 6 virtual cores)
Environment:
  JULIA_NUM_PRECOMPILE_TASKS = 10
  JULIA_DEPOT_PATH = /Users/vp/.julia
  JULIA_PKG_DEVDIR = /Users/vp/.julia/dev
```

```julia
[vp@MBP dev]:julia --startup-file=no
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.11.0 (2024-10-07)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using Pkg

julia> using REPL
ERROR: TypeError: 
Stacktrace:
  [1] active_module()
    @ REPL ~/.julia/juliaup/julia-1.11.0+0.aarch64.apple.darwin14/share/julia/stdlib/v1.11/REPL/src/REPL.jl:593
  [2] #invokelatest#2
    @ ./essentials.jl:1054 [inlined]
  [3] invokelatest
    @ ./essentials.jl:1051 [inlined]
  [4] active_module
    @ ./show.jl:519 [inlined]
  [5] 
SYSTEM (REPL): showing an error caused an error
ERROR: TypeError: 
Stacktrace:
  [1] active_module()
    @ REPL ~/.julia/juliaup/julia-1.11.0+0.aarch64.apple.darwin14/share/julia/stdlib/v1.11/REPL/src/REPL.jl:593
  [2] #invokelatest#2
    @ ./essentials.jl:1054 [inlined]
  [3] invokelatest
    @ ./essentials.jl:1051 [inlined]
  [4] active_module
    @ ./show.jl:519 [inlined]
  [5] 
SYSTEM (REPL): caught exception of type TypeError while trying to handle a nested exception; giving up

julia> versioninfo()
00000000), Char(0x00000000), Char(0x00000000), Char(0x68000000), Char(0x00000000), Char(0x6c000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000)], vals=09:33:43 [1829/1829]
ef2b7c0)[#<null>, #<null>, REPL.LineEdit.var"#254#262"(), #<null>, #<null>, #<null>, REPL.LineEdit.var"#254#262"(), #<null>, REPL.LineEdit.var"#254#262"(), #<null>, #<null>, #<null>, #<null>, #<null>, #<null>, #<null>], ndel=0, count=3, age=0x0000000000
000004, idxfloor=1, maxprobe=0),                                                                                                                                                                                                                             
  REPL.LineEdit.var"#253#261"(),                                                                                                                                                                                                                             
  #<null>,                                                                                                                                                                                                                                                   
  #<null>,                                                                                                                                                                                                                                                   
  REPL.LineEdit.var"#252#260"(),                                                                                                                                                                                                                             
  #<null>,                                                                                                                                                                                                                                                   
  #<null>,                                                                                                                                                                                                                                                   
  #<null>,                                                                                                                                                                                                                                                   
  #<null>,                                                                                                                                                                                                                                                   
  Base.Dict{Char, Any}(slots=Memory{UInt8}(16, 0x10eeefd70)[0x00, 0x00, 0xd0, 0x00, 0x00, 0x00, 0xd4, 0x00, 0xb5, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], keys=Memory{Char}(16, 0x10ee83bc0)[Char(0x00000000), Char(0x00000000), Char(0x7e000000), Char(0x
00000000), Char(0x00000000), Char(0x00000000), Char(0x68000000), Char(0x00000000), Char(0x6c000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000), Char(0x00000000)], vals=Memory{Any}(16, 0x10
ef2b0e0)[#<null>, #<null>, REPL.LineEdit.var"#254#262"(), #<null>, #<null>, #<null>, REPL.LineEdit.var"#254#262"(), #<null>, REPL.LineEdit.var"#254#262"(), #<null>, #<null>, #<null>, #<null>                                                                                                                                                                                                       
```

**Comment by gdalle** [comment]
Can you try the solution from here?

-  https://github.com/KristofferC/OhMyREPL.jl/issues/345#issuecomment-2397745411

**Comment by KristofferC** [comment]
We missed backporting https://github.com/JuliaLang/julia/pull/55316 it seems.

**Comment by vpuri3** [comment]
Great. I'll stick with 1.10 in the meantime

**Comment by ndgnuh** [comment]
This is probably the cause of this bug too, isn't it? (Loading some other packages like `IJulia` or `CairoMakie` also reproduces this)


```julia
julia> using Pluto
ERROR: InitError: MethodError: no method matching 
Stacktrace:
  [1] __init__()
    @ REPLExt ~/.julia/juliaup/julia-1.11.0+0.x64.linux.gnu/share/julia/stdlib/v1.11/Pkg/ext/REPLExt/REPLExt.jl:305
  [2] 
SYSTEM (REPL): showing an error caused an error
ERROR: TypeError: 
Stacktrace:
  [1] active_module()
    @ REPL ~/.julia/juliaup/julia-1.11.0+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/REPL.jl:593
  [2] #invokelatest#2
    @ ./essentials.jl:1054 [inlined]
  [3] invokelatest
    @ ./essentials.jl:1051 [inlined]
  [4] active_module
    @ ./show.jl:519 [inlined]
  [5] 
SYSTEM (REPL): caught exception of type TypeError while trying to handle a nested exception; giving up
```

**Comment by ufechner7** [comment]
Is this fixed on 11.1? If yes, can this bug be closed?
